docs/src/pages/components/stack/stack.md

```diff
@@ -0,0 +1,13 @@
+---
+title: React Stack component
+components: Stack
+githubLabel: 'component: Stack'
+---
+
+# Stack
+
+<p class="description">The Stack component manages children along the vertical and horizontal axes.</p>
+
+Stacks are used to align and distribute children along the vertical or horizontal axis.
+
+{{"demo": "pages/components/stack/InteractiveStack.js", "hideToolbar": true, "bg": true}}
```

Comment:
For the docs, maybe something in this order?:

```
## Usage

Something baseline, that shows the default vertical alignment.
Probably have to talk about the spacing.

## How it works

Maybe explain how it works and how it fits into the different alternative layout components?

## Direction

Something that shows the row direction.

## Responsive values

Maybe show how it can have responsive directions/spacing.

## Dividers

How to interspace React elements

## Playground

The current demo.
```

## Benchmark

- https://balance.reckon.com/package/stack
- https://arcade.design/docs/utilities/stack/
- https://chakra-ui.com/docs/layout/stack
- https://tailwindcss.com/docs/space
- https://react-ui.dev/components/Stack
- https://www.framer.com/api/stack/
- https://seek-oss.github.io/braid-design-system/components/Stack/